src/pages/elements/Service.js

```diff
@@ -247,6 +322,8 @@ function Service({ service, reports, documents }) {
           </details>
         </div>}
 
+      {<SaasFiles saas_files={ saas_files.filter(file => file['app']['name'] === service['name'])} settings={settings} />}
```

Comment:
You can reuse this https://github.com/app-sre/visual-qontract/blob/master/src/pages/elements/Service.js#L54-L61

Reply:
Updated!